misc/standard.talon

```diff
@@ -17,7 +17,10 @@ scroll up: edit.page_up()
 scroll down: edit.page_down()
 copy that: edit.copy()
 cut that: edit.cut()
-paste that: edit.paste()
+(pace | paste) that: edit.paste()
```

Comment:
I haven't had any issues with 'paste that', does pace help with your accent?

Reply:
yes